admin/app/components/solidus_admin/ui/table/toolbar/component.rb

```diff
@@ -1,14 +1,21 @@
 # frozen_string_literal: true
 
 class SolidusAdmin::UI::Table::Toolbar::Component < SolidusAdmin::BaseComponent
-  erb_template <<~ERB
-    <div class="
-      h-14 p-2 bg-white border-b border-gray-100
-      justify-start items-center gap-2
-      visible:flex hidden:hidden
-      rounded-t-lg
-    ">
-      <%= content %>
-    </div>
-  ERB
+  def initialize(**options)
+    @options = options
+  end
+
+  def call
+    tag.div(
+      content,
+      **@options.except(:class),
```

Comment:
optional nit, the class key would be overridden anyway, so it's technically unnecessary